### ISSUE TITLE
Excel leaderboard export fails

### ISSUE THREAD

**Issue by 16ajans** [MatrikMoon/TournamentAssistant#69]
Logged:
```
Mar 08 18:53:49 cvre01 TournamentAssistantCore[499911]: 18:53:49 App Command System.ArgumentException: The worksheet name cannot start or end with an apostrophe ('). (Parameter 'Name')
Mar 08 18:53:49 cvre01 TournamentAssistantCore[499911]:    at OfficeOpenXml.ExcelWorksheets.ValidateFixSheetName(String Name)
Mar 08 18:53:49 cvre01 TournamentAssistantCore[499911]:    at OfficeOpenXml.ExcelWorksheets.AddSheet(String Name, Boolean isChart, Nullable`1 chartType, ExcelPivotTable pivotTableSource, XmlElement sheetElement)
Mar 08 18:53:49 cvre01 TournamentAssistantCore[499911]:    at TournamentAssistantCore.Discord.Modules.QualifierModule.ExcelLeaderboardsAsync(String eventId)
Mar 08 18:53:49 cvre01 TournamentAssistantCore[499911]:    at Discord.Interactions.Builders.ModuleClassBuilder.<>c__DisplayClass11_0.<<CreateCallback>g__ExecuteCallback|1>d.MoveNext()
```

**Comment by 16ajans** [comment]
This can be circumvented by dumping the sqlite table in the meantime.

Also `nice`

**Comment by MatrikMoon** [comment]
Well that's... At least this'll be an easy one 🤣

**Comment by MatrikMoon** [comment]
Fixed in `a7c7afa95e729b716a1d13b256d9254f29bd4c54`! My bad for not making it a PR for easy linkability, but should be done on next release!
